packages/devextreme/js/__internal/ui/m_multi_view.ts

```diff
@@ -388,16 +403,18 @@ const MultiView = CollectionWidget.inherit({
     const { offset } = e;
     const swipeDirection = sign(offset) * this._getRTLSignCorrection();
 
-    _translator.move(this._$itemContainer, offset * this._itemWidth());
-
     if (swipeDirection !== this._swipeDirection) {
       this._swipeDirection = swipeDirection;
+    }
 
-      const selectedIndex = this.option('selectedIndex');
-      const newIndex = this._normalizeIndex(selectedIndex - swipeDirection);
+    const selectedIndex = this.option('selectedIndex');
+    const newIndex = this._normalizeIndex(selectedIndex - swipeDirection);
 
-      this._updateItems(selectedIndex, newIndex);
+    if (selectedIndex !== newIndex) {
```

Comment:
Actually if `selectedIndex === newIndex` there no any profit to call ` this._updateItems(selectedIndex, newIndex);` as well. So maybe just write
```javascript
if (selectedIndex === newIndex) {
    return;
}
```
to make no excess action?

Reply:
[eb9495e](https://github.com/DevExpress/DevExtreme/pull/28128/commits/eb9495e5a1192db390e67834c0ed4c2764c2b9d4)